src/main/java/io/jenkins/plugins/audit/helpers/Factory.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.audit.helpers;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class Factory {
```

Comment:
This class may be better named as something like `DateTimeUtil` or `DateTimeHelper`.